.test.env

```diff
@@ -0,0 +1,7 @@
+export SERVER_ENDPOINT=localhost:9000
+export ACCESS_KEY="masoud" 
+export SECRET_KEY="Strong#Pass#2022" 
+export ENABLE_HTTPS=1 
+export IGNORE_CERT_CHECK=0 
+export SSL_CERT_FILE=./tests/public.crt 
+export MINIO_CI_CD=true 
```

Comment:
We do not need anything like this.

Reply:
https://github.com/minio/minio-rs/pull/32#issuecomment-1578311770

Reply:
How you run your S3 server/service for local testing is out of scope of this project. You cannot use Github CI script locally. In fact, there is no need to test `minio-rs` locally. If a contributor does that, it is better to add respective functional tests rather than testing locally.

We need to keep the project tidy.